code/menuui/mainhallmenu.cpp

```diff
@@ -68,6 +68,8 @@ static main_hall_defines *Main_hall = nullptr;
 
 static int Main_hall_music_index = -1;
 
+bool Main_hall_poll_key = 1;
```

Comment:
yeah, `snazzy_menu_do` expects an int and since that's used in many places I opted to just follow convention here rather than inline static cast the bool. But I could go either way.

Reply:
wait. I'm dumb. I literally assigned it as a bool instead of an int. Ok. Changed.